src/aeromancy/action_runner.py

```diff
@@ -182,6 +188,7 @@ def run_actions(
         only: str | None,
         graph: bool,
         list_actions: bool,
+        tags: str | None,
```

Comment:
Can this be set[str] for consistency?  I also want 231 to go away since it doesn't trim "spaces, like, this " which could lead to subtle surprises.